wordpress-mobile/WordPress-iOS
# Today Widget - Default site

This PR separates the default site for the new (iOS 14) Today Widget from the existing one, because the latter might be changed later by users.

It also consolidates localizable strings all in one place and add a couple of localizations that were missing

Fixes #NA

To test:

- install the widget and make sure the site you see is your primary site.
- long press on the widget then "Edit Widget" and make sure it works as expected.

PR submission checklist:
